pkgs/racket-doc/scribblings/reference/equality.scrbl

```diff
@@ -747,6 +751,7 @@ and only access mutable data when the mode is true:
          (define p (rec (rotate-triple-paper self)))
          (define s (rec (rotate-triple-scissors self)))
          (hash-code-combine-unordered
+          (eq-hash-code struct:rotate-triple)
```

Comment:
This looks "wrong" to me, FWIW.

Reply:
Just a feeling? Or is there a hash collision that this allows, that could be avoided by doing something else?

Reply:
After some searching, I found this hash collision, involving `rotate-triple` and a `rotate-triple2` structure that's just an exact copy with only the name changed, to make
```racket
> (equal-hash-code (set (rotate-triple 'A 'B 'C) (rotate-triple2 'D 'E 'F)))
548402909030490366
> (equal-hash-code (set (rotate-triple2 'A 'B 'C) (rotate-triple 'D 'E 'F)))
548402909030490366
```

However... I personally don't think it's it's that important to fix this just for a documentation example.

Reply:
I write many programs by adapting examples in the documentation, so it would indeed be nice to fix the example. I can submit a separate PR to fix it though if you don't want to do it yourself.

```
     (hash-code-combine
      (eq-hash-code struct:rotate-triple)
      (hash-code-combine-unordered
       (hash-code-combine r p s)
       (hash-code-combine p s r)
       (hash-code-combine s r p)))
```

should do the job, right?

Reply:
I made the change... because after seeing your comment I also realized `rotate-triple` could get away with only
```racket
(hash-code-combine
 (eq-hash-code struct:rotate-triple)
 (hash-code-combine-unordered
  (hash-code-combine r p)
  (hash-code-combine p s)
  (hash-code-combine s r)))
```
and that simplification makes up for it